scss/core/forms.scss

```diff
@@ -72,7 +72,7 @@ select,
         border-width: 0 0 2px 2px;
         transform: rotate(-45deg);
         position: absolute;
-        z-index: 2;
+        z-index: map-get($fd-z-index-levels, "second");
```

Comment:
Still not sure why this forms element needs a z-index?  Is it to be displayed on top of something else?

Reply:
Yeah, maybe we can remove it. @saadmhybris what do you think?

Reply:
@ianquigley-sap @ckoutsiaris @saadmhybris ~ I believe that is the little dropdrown arrow that should be positioned on top of the select.